common/persistence/pinotVisibilityTripleManager.go

```diff
@@ -329,11 +329,16 @@ type userParameters struct {
 
 // For Pinot Migration uses. It will be a temporary usage
 // logUserQueryParameters will log user queries' parameters so that a comparator workflow can consume
-func (v *pinotVisibilityTripleManager) logUserQueryParameters(userParam userParameters, domain string) {
+func (v *pinotVisibilityTripleManager) logUserQueryParameters(userParam userParameters, domain string, override bool) {
 	// Don't log if it is not enabled
 	if !v.logCustomerQueryParameter(domain) {
```

Comment:
we can combine this with override check `if !v.logCustomerQueryParameter(domain) || override {}`

Reply:
Sure. I was thinking to make it more readable